examples/CSharp/CSharp.Examples.NUnit/Tests.cs

```diff
@@ -32,7 +32,7 @@ public void Test()
                     Simulation.KeepConcurrentScenarios(copiesCount: 1, during: TimeSpan.FromSeconds(2))
                 });
 
-            var nodeStats = NBomberRunner.RegisterScenarios(new[] {scenario}).RunTest();
+            var nodeStats = NBomberRunner.RegisterScenarios(new[] {scenario}).RunTest(Array.Empty<string>());
```

Comment:
I think we need to have only 2 public functions:
run: unit -> Result<NodeStats, string>
runWithArgs: string[] -> Result<NodeStats, string>

for C# we can use method overloading and basically have only 1 method Run. 

in function run: we need to understand the current environment: Console or Process.